go/pvl/interp.go

```diff
@@ -418,25 +425,29 @@ func validateScript(g proofContextExt, script *scriptT, service keybase1.ProofTy
 
 // Run each script on each TXT record of each domain.
 // Succeed if any succeed.
-func runDNS(g proofContextExt, userdomain string, scripts []scriptT, mknewstate stateMaker) []libkb.ProofError {
+func runDNS(g proofContextExt, userdomain string, scripts []scriptT, mknewstate stateMaker, sigIDMedium string) libkb.ProofError {
 	domains := []string{userdomain, "_keybase." + userdomain}
 	var errs []libkb.ProofError
 	for _, d := range domains {
 		debug(g, "Trying DNS for domain: %v", d)
 
-		err := runDNSOne(g, d, scripts, mknewstate)
+		err := runDNSOne(g, d, scripts, mknewstate, sigIDMedium)
 		if err != nil {
 			errs = append(errs, err)
 		} else {
 			return nil
 		}
 	}
 
-	return errs
+	// Return only the error for the first domain error
+	if len(errs) == 0 {
+		return nil
+	}
+	return errs[0]
```

Comment:
If the user meant to configure the second thing instead of the first, then the first error is probably going to be "doesn't exist" or whatever (which they already know), while the second error might be something more interesting. Would it be possible to combine them, maybe even with just string concatenation?